doc/IOTRACER.md

```diff
@@ -31,28 +31,24 @@ this instruction. Let's get a trace. Any time you can call the tool's help:
 
 ~~~{.sh}
 iotrace --help
-Usage: iotrace command [options...]
+Usage: iotrace [module] command [options...]
+
+Available modules:
+     -C    --trace-config                        Manages trace configuration
+     -M    --trace-management                    Manages available traces
+     -P    --trace-parsing                       Parses traces
 
 Available commands:
-           --get-fs-statistics                   Gets filesystem statistics
-           --get-latency-histogram               Gets latency histogram
-           --get-lba-histogram                   Returns a histogram of LBA hits
-           --get-trace-repository-path           Returns location of trace repository path
-     -T    --get-trace-statistics                Gets trace statistics
-     -G    --get-trace-summary                   Returns summary of specified trace
-     -H    --help                                Print help
-     -L    --list-traces                         Lists available traces
-     -P    --parse-trace                         Parses trace files to human readable form
-     -R    --remove-traces                       Removes specified trace(s)
-           --set-trace-repository-path           Sets location of trace repository path
-     -S    --start-trace                         Start IO tracing
-     -V    --version                             Print version
+     -H    --help                                Prints help
+     -S    --start-tracing                       Starts IO tracing
+     -V    --version                             Prints version
+
 ~~~
 
-We have to invoke _--start-trace_ command. Its help is: 
+We have to invoke _--start-tracing_ command. Its help is:
```

Comment:
This line sounds awkward. I would rephrase:

To start trace collection use --start-tracing command. Command usage: